ui/app/AppLayouts/Chat/ChatColumn/MessageComponents/NormalMessage.qml

```diff
@@ -18,7 +18,7 @@ Item {
 
     UserImage {
         id: chatImage
-        visible: (isMessage || isEmoji) && authorCurrentMsg != authorPrevMsg && !isCurrentUser
+        visible: (isMessage || isEmoji || isImage) && authorCurrentMsg != authorPrevMsg && !isCurrentUser
```

Comment:
We'll probably need a property to hold that condition since we use it a couple of times and it keeps growing with the different types.